lib/node-repo.js

```diff
@@ -149,11 +149,7 @@ function getBotPrLabels (options, cb) {
     page: 1,
     per_page: 100, // we probably won't hit this
     number: options.prId
-  }, (err, res) => {
-    if (err) {
-      return cb(err)
-    }
-
+  }).catch(cb).then(res => {
```

Comment:
Ditto: `.then(res => { ... }, cb)` or `.then(res => { ... }).catch(cb)` in case an error happens in the success callback